### PR TITLE
Correct spelling

### PR DESCRIPTION
**Pull request by az0**
Most changes were in comments, and there were a few changes to literals for log output.

There were no changes to variable names, function names, IDs, or functionality.

### CODE REVIEW

File: src/treelearner/feature_histogram.hpp [hunk 1]
```diff
@@ -359,7 +359,7 @@ class FeatureHistogram {
           continue;
         }
 
-        // mark to is splittable
+        // mark to is as able to be split
```
Comment (jameslamb) [inline src/treelearner/feature_histogram.hpp:362]:
```suggestion
        // mark as able to be split
```

File: src/treelearner/feature_histogram.hpp [hunk 2]
```diff
@@ -940,7 +940,7 @@ class FeatureHistogram {
           continue;
         }
 
-        // mark to is splittable
+        // mark to is able to be split
```
Comment (jameslamb) [inline src/treelearner/feature_histogram.hpp:943]:
```suggestion
        // mark as able to be split
```

File: src/treelearner/feature_histogram.hpp [hunk 3]
```diff
@@ -1033,7 +1033,7 @@ class FeatureHistogram {
           continue;
         }
 
-        // mark to is splittable
+        // mark to as able to be split
```
Comment (jameslamb) [inline src/treelearner/feature_histogram.hpp:1036]:
```suggestion
        // mark as able to be split
```

File: src/treelearner/gpu_tree_learner.cpp [hunk 4]
```diff
@@ -599,7 +599,7 @@ void GPUTreeLearner::BuildGPUKernels() {
     }
     histogram_kernels_[i] = program.create_kernel(kernel_name_);
 
-    // kernel with all features enabled, with elimited branches
+    // kernel with all features enabled, with eliminated branches
```
Comment (jameslamb) [inline src/treelearner/gpu_tree_learner.cpp:602]: I don't understand this part of the code, so I'm not actually sure if this was supposed to be "limited" or "eliminated".

I can't really tell based on the other code around this.

@shiyu1994 or @StrikerRUS could you help with a suggestion for this one?

**Review by jameslamb — CHANGES_REQUESTED** [review]
Thanks very much for taking the time to contribute to LightGBM.

I've left a few small suggested changes. You can accept them directly in the browser by clicking "commit suggestion".

**Comment by StrikerRUS** [comment]
Hmmm... Something new.

image

**Comment by jameslamb** [comment]
> Hmmm... Something new.
> 
> image

that is because of abuse by crypto miners: https://www.bleepingcomputer.com/news/security/github-actions-being-actively-abused-to-mine-cryptocurrency-on-github-servers/

**Comment by StrikerRUS** [comment]
> that is because of abuse by crypto miners

Ah, I see! Good protection. I remember I saw 2 attacking PRs in my another repo several weeks ago.

I'm clicking "Approve and run".

File: src/treelearner/gpu_tree_learner.cpp (hunk 4 above)
Comment (az0) [reply src/treelearner/gpu_tree_learner.cpp:602]: I am totally not an expert on this code, but in `\src\treelearner\ocl\histogram16.cl` on lines 346 there is a related statement. 

```
#if ENABLE_ALL_FEATURES == 1
    // hopefully the compiler will propogate the constants and eliminate all branches
```

So I think "eliminated" is correct

Comment (jameslamb) [reply src/treelearner/gpu_tree_learner.cpp:602]: ah, nice! Ok yes that makes sense, thank you

**Review by jameslamb — APPROVED** [review]
Looks good to me, thanks very much!

**Comment by jameslamb** [comment]
Thanks very much @az0 ! Come back and contribute any time :grinning:

**Comment by az0** [comment]
> Thanks very much @az0 ! Come back and contribute any time 😀

@jameslamb 

You're welcome. Thank you for making it easy and quick.

**Pull request merged**